dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetVersion.java

```diff
@@ -607,7 +607,9 @@ public boolean equals(Object object) {
 
     @Override
     public String toString() {
-        return "[DatasetVersion id:" + getId() + "]";
+        String id = getId() == null ? "id doesn't exists" : getId().toString();
+
+        return "[DatasetVersion id:" + id + "]";
```

Comment:
Not sure why bother with defining special string for null. Does `id:null` is not self explanatory?